### PR TITLE
Update information-barriers-policies.md

### PR DESCRIPTION
**Pull request by Gautamkv**
Hide from address list enabled set to false for users in same IB segment to join Teams meeting.

### CODE REVIEW

File: microsoft-365/compliance/information-barriers-policies.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ In addition to the [required licenses and permissions](information-barriers.md#r
...
+- **No address book policies**: Before you define and apply information barrier policies, make sure no Exchange address book policies are in place. Information barriers are based on address book policies, but the two kinds of policies are not compatible. If you do have such policies, make sure to [remove your address book policies](/exchange/address-books/address-book-policies/remove-an-address-book-policy) first. Once information barrier policies are enabled and you have hierarchical address book enabled, all users ***who are not included*** in an information barrier segment will see the [hierarchical address book](/exchange/address-books/hierarchical-address-books/hierarchical-address-books) in Exchange online. Additionall ensure the user is visible in the address list with the HiddenFromAddressListsEnabled property set to $False.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md]:
```suggestion
- **No address book policies**: Before you define and apply information barrier policies, make sure no Exchange address book policies are in place. Information barriers are based on address book policies, but the two kinds of policies are not compatible. If you do have such policies, make sure to [remove your address book policies](/exchange/address-books/address-book-policies/remove-an-address-book-policy) first. Once information barrier policies are enabled and you have hierarchical address book enabled, all users ***who are not included*** in an information barrier segment will see the [hierarchical address book](/exchange/address-books/hierarchical-address-books/hierarchical-address-books) in Exchange online. Additionally, ensure that the user is visible in the address list with the **HiddenFromAddressListsEnabled** property set to $False.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Gautamkv, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@Gautamkv Gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by robmazz** [comment]
@Gautamkv - thanks for the feedback. This topic is undergoing changes for the newer version of IB, I will discuss this the PG and confirm that this addition is still accurate. Thanks again.

**Pull request closed without merging**